tests/Shared/RepoTesting/Aspire.Testing.Repo.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup Condition="'$(TestsRunningOutsideOfRepo)' == 'true'">
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <!-- IsAspireHost property might not be available here, so allow infering that from the project name -->
+  <ImportGroup Condition="'$(RepoRoot)' != 'null' and '$(TestsRunningOutsideOfRepo)' != 'true' and ($(MSBuildProjectName.EndsWith('AppHost')) or '$(IsAspireHost)' == 'true')">
+    <Import Project="$(RepoRoot)src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.props" Condition="Exists('$(RepoRoot)src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.props')" />
+  </ImportGroup>
+
+ <PropertyGroup>
+    <SharedDir Condition="'$(SharedDir)' != ''">$([MSBuild]::EnsureTrailingSlash($(SharedDir)))</SharedDir>
```

Comment:
Can we make it so that whoever defined SharedDir to ensure the trailing slash? We usually don't do this for directory properties. We just assumed whoever set it did it right.